pkg/controller/node/nodecontroller.go

```diff
@@ -243,15 +243,10 @@ func (nc *NodeController) Run(period time.Duration) {
 		nc.evictorLock.Lock()
 		defer nc.evictorLock.Unlock()
 		nc.podEvictor.Try(func(value TimedValue) (bool, time.Duration) {
-			remaining, err := nc.deletePods(value.Value)
-			if err != nil {
-				utilruntime.HandleError(fmt.Errorf("unable to evict node %q: %v", value.Value, err))
+			if err := nc.evictPodsNoRateLimit(value.Value); err != nil {
```

Comment:
The name of the function is misleading - this call is rate limited, as 'Try' is. Please rename it to plain 'evictPods' or something like that (actually we need some better naming for all those functions - having `deletePods`, `terminatePods` and `evictPods` is not very readable, but I'll handle this separately).